packages/mobile/src/pages/chat/chat-page.tsx

```diff
@@ -509,11 +531,6 @@ const ChatPage = memo(({ route, navigation }: ChatProp) => {
     return <NoConversationHeader title="No conversation found" goBack={doReturnToConversations} />;
   }
 
-  // If there are assets, show the file overview; Not when it's audio only, as we send that directly
-  if (assets?.length && !(assets.length === 1 && assets[0].type?.startsWith('audio/'))) {
```

Comment:
Any reason you made the asset overview into it's own page and navigate into it? Every navigation adds a performance hit.. Which can be worth it if it's an "actual navigation".. But in this case, it would always get back to the chat page?

Reply:
Multiple Reasons:

I didn't like the approach of handling assets inside the Chat Page itself. We had it before cause we used to show mini previews above the composer before creating a separate page/component. 

With the creation of this route, this helps to call this page from anywhere that only deals with assets. 
Secondly it lightens the JS thread as it doesn't call all the useChatMessages and conversation hooks when you are only dealing with sending messages (if invoked from the Share page or any external route)
Third: It helps in support for sending to multiple recipients with a custom message you want to send
Fourth: UX. Converting to a route gives a slide up animation that feels consistent with other apps

Reply:
Apart from not liking which to be fair is somewhat personal opinion...

1, 2 and 3 are all solved by just re-using the `ChatFileOverview` component right? You didn't need a new route to share a component;

I can agree with 4.. But it includes the perfomance hit of navigating with it just for not building the animation again so we can re-use the navigation animation

Reply:
We could technically reuse it everywhere but I still don't understand how it would cause a performance hit, since its inside the same nested navigator and all the previous screen are just being replaced instead of push to save memory (except when invoked from chat screen) ?

Reply:
Well I'm specifically talking about adding assets from inside a conversation already. I'm definitely not saying that shouldn't have made it into a route, just doubting the fact that you needed to navigate to that same route from within a ChatScreen

So in that case when it's navigating at a higher level, it will render all navigation related components. So instead of just rendering <ChatFileOverview /> when there's assets, you navigate in which the navigation adds overhead and eventually renders <ChatFileOerview />